app/customer.py

```diff
@@ -0,0 +1,46 @@
+from math import dist
+from typing import List
+from app.shop import Shop
+
+
+class Customer:
+    def __init__(self, customer: dict) -> None:
+        self.name: str = customer["name"]
+        self.product_cart: dict = customer["product_cart"]
+        self.location: list = customer["location"]

```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]